src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2789,6 +2789,37 @@ protected override IPointerTypeSymbol CommonCreatePointerTypeSymbol(ITypeSymbol
             return CreatePointerTypeSymbol(elementType.EnsureCSharpSymbolOrNull<ITypeSymbol, TypeSymbol>("elementType"));
         }
 
+        protected override INamedTypeSymbol CommonCreateTupleTypeSymbol(ITypeSymbol[] elementTypes, string[] elementNames)
```

Comment:
I do not see a single test for this function. At the very least every branch should be covered with a test.